### PR TITLE
update deploy script

### PR DESCRIPTION
**Pull request by ebidel**
R: all

### CODE REVIEW

File: scripts/deploy_site.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ compass compile
 
 # Run jekyll and build api docs.
 #grunt docs # grunt docs doesn't work on jekyll 2.0
-grunt vulcanize
+grunt vulcanize-elements
```
Comment (arthurevans) [inline scripts/deploy_site.sh]: Why are we changing the target name?

**Comment by robdodson** [comment]
why not replace both lines with `grunt docs`. That will do that plus run doc-merge

**Comment by ebidel** [comment]
Done.

**Comment by ebidel** [comment]
PTAL

**Comment by robdodson** [comment]
LGTM :dancer:

**Pull request merged**